plugins/octant/cmd/antrea-octant-plugin/traceflow.go

```diff
@@ -404,15 +340,15 @@ func (a *traceflowPlugin) getTfRows() []component.TableRow {
 	tfRows := make([]component.TableRow, 0)
 	for _, tf := range tfs.Items {
 		tfRows = append(tfRows, component.TableRow{
-			tfNameCol:       component.NewText(tf.Name),
+			tfNameCol:       component.NewLink(tf.Name, tf.Name, octantTraceflowCRDPath+tf.Name),
 			srcNamespaceCol: component.NewText(tf.Spec.Source.Namespace),
 			srcPodCol:       component.NewText(tf.Spec.Source.Pod),
 			dstNamespaceCol: component.NewText(tf.Spec.Destination.Namespace),
 			dstPodCol:       component.NewText(tf.Spec.Destination.Pod),
-			crdCol:          component.NewLink(tf.Name, tf.Name, octantTraceflowCRDPath+tf.Name),
 			phaseCol:        component.NewText(string(tf.Status.Phase)),
 			ageCol:          component.NewTimestamp(tf.CreationTimestamp.Time),
 		})
 	}
-	return tfRows
+	tfCols := component.NewTableCols(tfNameCol, srcNamespaceCol, srcPodCol, dstNamespaceCol, dstPodCol, phaseCol, ageCol)
+	return component.NewTableWithRows("Traceflow List", "We couldn't find any traceflows!", tfCols, tfRows)
 }
```

Comment:
Better to make the titles consistent, like "Traceflow Info", as this one is shown in the overview page and the other two tables are called "Antrea Controller Info" and "Antrea Agent Info".

Reply:
Updated